### ISSUE TITLE
Releasedate

### ISSUE THREAD

**Issue by greenhalos** [qoomon/maven-git-versioning-extension#18]
Hey,

I am currently waiting for a new release to use the commit specific configurations. Will there be a release soon?

**Comment by qoomon** [comment]
Which feature do you mean?

**Comment by greenhalos** [comment]
The latest released version in maven central is 3.0.2.

The following configuration is not valid with this release:
```xml
<configuration>
    <branches>
        <branch>
            <pattern>master</pattern>
            <versionFormat>${version.release}</versionFormat>
        </branch>
    </branches>
    </tags>
    <commit>
        <versionFormat>${commit}</versionFormat>
    </commit>
</configuration>
```
I get the following error:
```
Element 'commit' does not have a match in class me.qoomon.maven.extension.gitversioning.config.model.Configuration at line 13
```

**Comment by qoomon** [comment]
Oh my bad, I will release it tomorrow.

**Comment by qoomon** [comment]
just released version `3.1.0`, it may take a while to sync to maven central.
https://search.maven.org/artifact/me.qoomon/maven-git-versioning-extension/3.1.0/maven-plugin
